### PR TITLE
Adds the Central Inspector role

### PR DESCRIPTION
**Pull request by joshepvodka**
## About the PR

Well now I feel silly but PJB convinced me it's not a good idea. I'll be keeping this open if the maintainers want this added as a admin or ghost role.
Thanks for (mostly) everyone that was patient enough to discuss this PR.

This PR adds the role of the Central Inspector, based on the Internal Affairs Agent job from Paradise Station. Also adds assets relating to the role, such as a stamp, PDA, ID card, job icon, spawner and humanoid spawner.

### Central Inspector
You're an Internal Affairs Agent sent from CentCom to aid in managing the station. Your duties include:
- Ensuring crew work in good conditions.
- Ensuring Space Law is followed by security personnel and command crew.
- Make sure all departments operate in accordance to SOP.
- Attempt mediating diplomatically any issues that arise between departments or commanders.
- Inform commanders of personnel or structural problems happening in the station so they can take action.

You answer directly to CentCom, but while on the station you are under the rule of the captain and the departmental heads. You hold no power over any crew members. Your duties relate only to diplomacy, not administration, this means you are not allowed to demote crew or choose what actions are taken in any department, instead, report issues and give advice (if you're welcome to do so).
The role requires:
- 10 hours as Captain
- 10 hours as HoS
- 45 hours in Command

The role is basically roleplay only, you serve no real function besides talking to people and writing on paper.

## Why / Balance

I think it's a good addition to the game because not only it is very fun for players that like following rules and engaging on paperwork and roleplaying, but also serves to correct actions taken by command and security that breach Space Law or SOP. As a player that usually plays Sec/Command/Lawyer I often see wardens not doing their job, security giving wrong sentences or taking too long to give out a sentence to their prisoners, HoS taking charge of the station when captain dies, instead of HoP, management issues in many different sectors, demotion worthy offenses not being reported or incentivized to be punished, crew simply not doing their job or NRPing, heads going rogue, etc. Also I think an IAA or Inspector of some sort would greatly aid command in running the station, and also would add more roleplay opportunities in every gamemode.

## Technical details

The role time requirements are so:
Captain: The player learns how the station is ran by the captain and what their responsibilities are.
HoS: How security should act during the shift and how space law should be carried out properly.
Extra command time: So players spend another 10 hours after doing the captain and HoS requirements playing other command roles to get a better understanding of the other department heads.

The CentCom look is an aesthetic choice, but I think it also adds fun new roleplay implications, as CC officials are rarely seen in the station.

You have basic access to all departments. Those are: access to bridge, maints, brig, service, engi, science, medical and cargo (salvage not included)

The Central Inspector spawns in with a CC headset that includes the CentCom encryption key, as an employee from CC I think it makes sense for them to be able to talk with ERT squads, Central Commanders or even the Death Squad in case they are ever called (probably by the CI). Tell me if I should remove it.

I think that as an inspector the player could also aid the admin, in reporting bad behaviour and faxing for events such as ERT, CBURN, Death Squad or maybe a visit from a CentCom Official.

Currently the Central Inspector stamp spawns in their backpack because as of yet no map has an office for them, so it's a temporary measure.

This PR includes assets made for PRs #22700(dark green suit) and #22693(CentCom folder and clipboard).
Basically everything required to add the role is already in this PR. (It's still missing time requirements to unlock the role, as this should be discussed.)

## Media

image
image
Starting Equipment 
image
image
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Applications for Central Inspector are now open! Talk to HR in CentCom to apply!

### CODE REVIEW

**Comment by Redfire1331** [comment]
add a function to ticket the station that shows the at the end of the round all the tickets the station got slapped with

**Comment by brainfood1183** [comment]
I can see the role getting added, but i think it shouldnt be contantly in rotation but an event or admeme only.

**Comment by joshepvodka** [comment]
I was aiming to make a role available from roundstart. If i remove the CentCom references could it be added as a normal Internal Affairs Agent?

**Comment by Holinka4ever** [comment]
I will higly recommend not to merge it, cause IAA job role doesnt work in lrp.

I have a plenty arguments why IAA is just a lawyer with command access... wait why you gave them hop access without hop. By the way it sounds powergamey, even in place where i play, IAA have only command, brig and maints access.

Thing is that IAA is an event role, not a starting job, because their actions are linked to mods whom have a mood to work with them, and if there no mods who are interesting, its just a lawyer.

BUT, giving them hop access is sounds so weirdly in a lot of points. They can just powergame all usefull stuff as sec armour and helmet, insulas, axes, chemicals. Huh, these sound ridiculuos.

**Comment by joshepvodka** [comment]
> I have a plenty arguments why IAA is just a lawyer with command access... wait why you gave them hop access without hop. By the way it sounds powergamey, even in place where i play, IAA have only command, brig and maints access.
> 
> BUT, giving them hop access is sounds so weirdly in a lot of points. They can just powergame all usefull stuff as sec armour and helmet, insulas, axes, chemicals. Huh, these sound ridiculuos.

I didn't even think of it this way. I gave them access to plenty of places so they can perform their duty of talking to people. Also, their job just doesn't involve armor, insuls, axes or chemicals. It's a command role, if the player acts like a tider, they should probably get role banned. I don't see HoPs going into sec to get armor vests or CEs carrying axes on their back from roundstart either.

>Thing is that IAA is an event role, not a starting job, because their actions are linked to mods whom have a mood to work with them, and if there no mods who are interesting, its just a lawyer.

I'm very sure they are a roundstart role in Paradise Station, which is where I took inspiration from. A lawyer's job also isn't to help heads manage the station or follow SOP, it's to make sure security doesn't treat prisoners badly.

**Comment by Holinka4ever** [comment]
>  I didn't even think of it this way. I gave them access to plenty of places so they can perform their duty of talking to people.

They can talk to them outside having departament access, in hall, bar, plenty places without involving departament restricted places. But if you have other point of view, sure, share with all of us.


> Also, their job just doesn't involve armor, insuls, axes or chemicals. It's a command role, if the player acts like a tider, they should probably get role banned. 

Human factor, and people powergaming desires. I can simply say that if a pep have an egineering access without being engineer, he will get soon insuals and belt. Same with sec. By the way, its a job role that doesnt have high risk item, or being head of departament, being the most safe role cause antags will rarely touch you. Except if antags hunts for access or rdm.

>  I don't see HoPs going into sec to get armor vests or CEs carrying axes on their back from roundstart either.

Oh they do,  if you dont see them, or you wished not to see them at all, it doesnt mean that they dont steal other departaments, by the way CE has a right to do it. But hop breaching into sec room to take armour and baton is common view.

> I'm very sure they are a roundstart role in Paradise Station, which is where I took inspiration from. A lawyer's job also isn't to help heads manage the station or follow SOP, it's to make sure security doesn't treat prisoners badly.

Paradise station has own stones in wheat field, one of them heavenly involving mods into game(where they shouldnt do it). Because, if sec doesnt listen to IAA that they do things wrong, IAA just involve mods to role ban a pep. Thats all is what an IAA.

**Comment by Nairodian** [comment]
> I didn't even think of it this way. I gave them access to plenty of places so they can perform their duty of talking to people. Also, their job just doesn't involve armor, insuls, axes or chemicals. It's a command role, if the player acts like a tider, they should probably get role banned. I don't see HoPs going into sec to get armor vests or CEs carrying axes on their back from roundstart either.

I'm not sure what I feel about the role in it's entirety, but I'd personally cut it down to just the base line access if all it's there for is to talk to people. Something like, Command, Service, Maintenance, Engineering, Research, Brig, Lawyer(?), Cargo, and Medical.

**Comment by Holinka4ever** [comment]
> > I didn't even think of it this way. I gave them access to plenty of places so they can perform their duty of talking to people. Also, their job just doesn't involve armor, insuls, axes or chemicals. It's a command role, if the player acts like a tider, they should probably get role banned. I don't see HoPs going into sec to get armor vests or CEs carrying axes on their back from roundstart either.
> 
> I'm not sure what I feel about the role in it's entirety, but I'd personally cut it down to just the base line access if all it's there for is to talk to people. Something like, Command, Service, Maintenance, Engineering, Research, Brig, Lawyer(?), Cargo, and Medical.

To big amount of access, plenty of them to be sure. They still can tide things from restricted vendomats. Steal meds from medical and other.
If they want to talk, they or can try use command radio to call head of departament or just call a someone,  try to catch scientists or engineers, try talk with medical team cause they are always open. 
In place where i play, they only have command,  brig and maints access. And its pretty fair.

In my opinion, they dont need these access to just chit-chat, its a weirdly wrong.

**Comment by JoeHammad1844** [comment]
this should probably be whitelist only

**Comment by nikthechampiongr** [comment]
Hello I'll just leave the comments I told you on discord here. As implemented in this PR the inspector is an supercharged lawyer with extended access to every department. They have really abstract responsibilities that heavily conflict with those of the captain, heads of departments, and lawyers. I'm especially not sure something like this would work on lrp, it would probably keep causing power struggles between command and the inspector. The access can be fixed easily, although the conflicting responsibilities and the effects it would have on lrp would need the inspector's role to be completely rethinked. The sprites tho look great.

**Comment by lzk228** [comment]
while wizden is mostly lrp and SOPs is just a one page, i dont think any roles like that its a good idea

> this should probably be whitelist only

no, just mrp at least and it'll be good

**Comment by Golinth** [comment]
Personally, I love this. Whether it be an in-game culture, or another reason, I often see lawyers ignored despite having legitimate issues or claims. I also think along with this, there should be a "space law-lite" to go into LRP, but that is just wishful thinking on my part.

**Comment by nikthechampiongr** [comment]
> there should be a "space law-lite" to go into LRP, but that is just wishful thinking on my part.

An admin is working on this I believe. Haven't heard news about it in a while tho.

**Comment by joshepvodka** [comment]
According to feedback the CI shouldn't have the same level of access as the HoP. I adjusted the job's access to include only basic access to all departments, those are:
  - Command
  - Service
  - Maintenance
  - External
  - Engineering
  - Research
  - Brig
  - Cargo
  - Medical

**Comment by joshepvodka** [comment]
> while wizden is mostly lrp and SOPs is just a one page, i dont think any roles like that its a good idea

I think thats very true for the SOP situation. JarlElbow, the person that first suggested the role, is working on writing an expanded SOP. If they give up on it I'll give expanding it a try.

**Comment by Holinka4ever** [comment]
> According to feedback the CI shouldn't have the same level of access as the HoP. I adjusted the job's access to include only basic access to all departments, those are:
> 
> * Command
> * Service
> * Maintenance
> * Engineering
> * Research
> * Brig
> * Cargo
> * Medical



Just again, why they need these access? 
To just a chit-chat?
I am neutral and friendly with command, maints and brig access, but others are:
In Engineering peps will munch insulas and toolbelts, in research they gonna use protoalthe to make stuff for themself, in cargo they will try to order things and in medical they cant steal medicine.Also service, they are paperworkers, nor a swiss blade.

Dont tell me that they not gonna do it, dont tell that IAA players will be crystal clear, it will turn them powergamey.

And if need "inspect" departament, just ask worker from there, its more friendly rather than just rush in to because you have access. Its a rude that you have a keys from house of you neighbours, its really wrong way of "mrp" as you trying to achive.

**Comment by joshepvodka** [comment]
Every single role in the game can have powergamers. You can always ahelp people that are going out of their way to get equipment they don't need, especially if they have command roles. Basic access to all departments are staying unless more people disagree with it and have good arguments to back up their disagreement.

**Comment by Aisu9** [comment]
i'm afraid implementing this role with the current community and role-playing issue is not the best thing to do right now.

**Comment by S1ss3l** [comment]
I feel like the fear and misconception about what powergaming is will heavily hinder the game developpement.

it's a command role that is designed for rp. If player don't wanna engage in that and instead turn to tider shittery, they should be rolebanned for it, like any other command role with more serious responsability. 

Engineer having engi access don't go rogue every round munching up half a station's worth of hv cable just cause they got insuls, they engage in the core mechanic of their role. 

Slapping a mrp requirement and a job time requirement (maybe some times spent as captain) should go a long way into making sure only motivated people hop into that role. Also preventing them from antag maybe ?

**Comment by joshepvodka** [comment]
>Slapping a mrp requirement and a job time requirement (maybe some times spent as captain) should go a long way into making sure only motivated people hop into that role. Also preventing them from antag maybe ?

I was thinking of giving them a Captain and Command time requirement so only people that actually know how command works are allowed to play it, after all it's a role that's supposed to be even more trusted than the captain, as they are meant to police all of command.
Also inspectors can't be antagonists and they spawn in mindshielded.

**Comment by Aisu9** [comment]
> I feel like the fear and misconception about what powergaming is will heavily hinder the game developpement.
> 
> it's a command role that is designed for rp. If player don't wanna engage in that and instead turn to tider shittery, they should be rolebanned for it, like any other command role with more serious responsability.
> 
> Engineer having engi access don't go rogue every round munching up half a station's worth of hv cable just cause they got insuls, they engage in the core mechanic of their role.
> 
> Slapping a mrp requirement and a job time requirement (maybe some times spent as captain) should go a long way into making sure only motivated people hop into that role. Also preventing them from antag maybe ?

one of the issue is MRP is turning into LRP/NRP because of the current status of lizard, people come from a media were no one is roleplaying and do what **they** want, not what the game want you to do. the issue has been known for something like a year and nothing succesful was tried up until recently one of the admins snaped out after experiencing first hand the shittery of Lizard. there is absolutely no way this role here will be use for the intended purpose and will be abused even on MRP.

**Comment by S1ss3l** [comment]
> one of the issue is MRP is turning into LRP/NRP because of the current status of lizard, people come from a media were no one is roleplaying and do what **they** want, not what the game want you to do. the issue has been known for something like a year and nothing succesful was tried up until recently one of the admins snaped out after experiencing first hand the shittery of Lizard. there is absolutely no way this role here will be use for the intended purpose and will be abused even on MRP.

So you suggest we now balance the game around that fact ? I think this concern is outside the scope of this pr, it's the game job's to provide a mrp-friendly environnement, rich in content, and the admin's job to enforce the rules to ensure people play accordingly.

Again, I really think a captain and command time requirement like @joshepvodka suggested would go a long way into making sure only truly invested people hop into that role. 

If tider are willing to go through the ordeal of playing every departement enough to unlock the head, playing each one enough to unlock the captain and playing the cap enough to unlock the inspector, all the while not being a little shit and being whitelisted, I don't think they are a tider anymore and can be trusted with such a role.

Keeping it behind cap role requirement already nullify most powergaming argument imo, if they wanted to abuse a role, it'd be a head or the captain.

(I want to make clear that I got absolutely no beef with the admin team that, as far as I'm concerned, have always been stellar in how they handle the game and have made many unforgettable moment for me. They have a lot of work and this may pill up on top of it but my point is at the end of the day it's still their purview to enforce the MRP aspect of the game)

**Comment by PJB3005** [comment]
TL;DR this should just be an alt job title/loadout for Lawyer. Which is quite literally what the **Internal Affairs Agent** is on some SS13 servers.

Long read:

This should just be an alt job title/loadout for Lawyer. Which is quite literally what the **Internal Affairs Agent** is on some SS13 servers. The job description here is very much "guy in a suit, has a briefcase, is going to talk to you about Procedure(tm)". Which is a description that fits just as much with Lawyer.

This job is pure RP. I don't have anything against that, giving people jobs that are "just RP" so they do something creative themselves is good, but this job does not have a significant enough function that it meaningfully needs to be separate from Lawyer. Or, indeed, have the amount of access and power as proposed here. I'll deconstruct that more here.

> Your duties include:

This is all HRP nonsense that nobody[^1] cares about on short SS14 rounds. Involving anybody with this stuff is at best going to annoy them, at worst going to ruin their round. Nobody is playing engineering and wants to have a lawyer force their way into the department to talk about "safety standards". Not a single person. If an annoying clown decides to start shitting up medbay they're easy to remove. An IAA that manages to get the captain to listen to their RP can ruin people's rounds by "doing their job".

I know from first hand experience because I've had IAA do that to me before to an LRP SS13 server. Personally I thought it was a unique change of pace and went along (even if I was fucking furious the whole time). The doctor the IAA made his "victim" for an hour was not so amused however and wanted to file a player complaint due to a ruined round, which I fully understood.

> You answer directly to CentCom, but while on the station you are under the rule of the captain and the departmental heads. You hold no power over any crew members. Your duties relate only to diplomacy, not administration, this means you are not allowed to demote crew or choose what actions are taken in any department, instead, report issues and give advice (if you're welcome to do so).

> [...] but also serves to correct actions taken by command and security that breach Space Law or SOP. As a player that usually plays Sec/Command/Lawyer I often see wardens not doing their job, security giving wrong sentences or taking too long to give out a sentence to their prisoners, HoS taking charge of the station when captain dies, instead of HoP, management issues in many different sectors, demotion worthy offenses not being reported or incentivized to be punished, crew simply not doing their job or NRPing, heads going rogue, etc.

So for starters I'm gonna repeat here that *the responsibilities overlap with lawyer*. But aside from what, what makes you think that suddenly security or command is going to respect the CI's opinion when they clearly didn't ? The fact that they have more access? That's not going to do anything.

The fact of the matter is that people don't respect lawyers because it's primarily an LRP game and nobody cares about The Procedure. Giving the Procedure guy more access isn't going to make people care.

> You have basic access to all departments. Those are: access to bridge, maints, brig, service, engi, science, medical and cargo (salvage not included)

They absolutely do not need this access. The powergame aspect of this has already been stated by others, but even from the RP side this makes no sense: somebody with a suit should not be able to just waltz on into medbay (sterile environment), engineering (heavy machinery) or science (both). You wanna inspect a department? You get somebody qualified from the department to escort you. (we both know you're never gonna get anybody to let you in on LRP.)

> it's a command role that is designed for rp. If player don't wanna engage in that and instead turn to tider shittery, they should be rolebanned for it, like any other command role with more serious responsability.

This is not how basic game design or player psychology works. Giving people more access than they need *will* mean they will *help themselves* when they believe there is a "justified" time. AI's rogue? I'm just gonna help myself to insuls. It's not powergaming because I didn't do it in advance right?

Policing the issues created by giving them too much with "they should be role banned if they abuse it" (more admin load) in an LRP game is a terrible solution. The fact of the matter is that if we give them that access, under LRP rules they are not acting out of line if they help themselves to insuls under the prior justification.

> Again, I really think a captain and command time requirement like @joshepvodka suggested would go a long way into making sure only truly invested people hop into that role.
> 
> [...]

This entire section is implying that somehow having high playtime requirement filters out bad players and means you only get people who Really Wanna RP Properly. Yeah I don't buy it chief. The fact of the matter is that you're adding a *really powerful* role slot to the game, which somebody can take if the other top two (cap and HoP) are taken.

**In fact** it almost reads like CI is the *end all be all* role due to the playtime requirements and description: you can do *whatever you want!* The perfect balance of *no responsibilities* with *very flexible and universal access* cannot go wrong. Waltz in anywhere under the guise of "inspection", while having no disk to protect when it comes down to it.




[^1]: Salamander players: you can still do this just fine as a merged-role lawyer and also I didn't ask.

**Comment by PJB3005** [comment]
Meme:

The myth of "consensual RP"

Clown on operating table: I consent
Surgeon removing his limbs: I consent

The IAA: I don't!

Is there somebody you forgot to ask?

**Comment by PJB3005** [comment]
One more comment, because I forgot to include it in the body of that wall of text up there: 

I cannot help but draw parallels to Senior roles, which were removed with good reason:

* More access than normal
* High playtime requirement in an effort to filter "bad players"
* Stated function is primarily an RP function that can be executed regardless of the "added prestige" of the base job.

**Comment by Nairodian** [comment]
Yeah the above pretty much sums up all my thoughts. It probably should be admin-only (if at all), if an admin chooses to use it, they'll actually be able to give the role a goal, likely because something was violated already, and be able to watch them as to ensure they don't abuse their rank and access.

This is not something I'd consider making round-start on LRP, MRP, and hell, probably not HRP either. Command members should  and are trusted to enforce the SOP, Law, etc, themselves.

**Comment by Nairodian** [comment]
I think it's also been stated a few times in a slew of places that anything Central Command should stay admin-only as many people consider it admin intervention, and I think it should probably stay that way.

**Comment by DummeHytteOst** [comment]
isn't this just lawyer, but more access, and better headset?

i don't see it being a good role, if that is the case.

**Comment by joshepvodka** [comment]
Well, now even I can't see the role working as part of the game... how silly.
I'll keep the PR open in case maintainers want to keep it as an admin/ghost role, so at least something can be kept from it. In that case tell me so I can remove unnecessary stuff about it.

Loadouts when?
